Sources/RswiftCore/Util/Struct+InternalProperties.swift

```diff
@@ -64,6 +64,7 @@ extension Struct {
         name: "localeBundle",
         generics: nil,
         parameters: [
+          .init(name: "parentBundle", type: Type._Bundle),
```

Comment:
When there are multiple variables of the same type in the same namespace, I prefer to give all of them more detailed names to prevent possible wrong usage in the future when code is changed/refactored.